.releaserc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "branches": [
+    "master",
+    "next"
+  ],
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    "@semantic-release/release-notes-generator",
+    "@semantic-release/github",
+    "@semantic-release/npm"
+  ]
+}
```

Comment:
would it be difficult to let master branch be the HEAD and have a different branch, e.g. released be the branch that we use for release?